### DISCUSSION TITLE
Computation of Interventions Avoided

### DISCUSSION THREAD

**Discussion by alkat19** [MSKCC-Epi-Bio/decisioncurveanalysis#16] [Q&A] [answered]
Hey there and thank you for that excellent package.

I have a question regarding the interventions avoided computation. What formula is being used to compute it and what's the intuition behind it? Does it make use of the TNs and FNs or is it a comparison of net benefits across different strategies?

**Comment by shaunporwal** [comment, marked as answer]
Hi Alexandros,

The calculation for the net interventions avoided is:

net_intervention_avoided = (net_benefit - net_benefit_all) / (threshold / (1 - threshold)) * nper

net_benefit = net benefit of using a particular strategy
net_benefit_all = net benefit of treating everyone
threshold = probability of a disease or event at which a patient would opt for treatment
nper = scaling factor for # of interventions, default is 1

The calculation of interventions avoided doesn't use TNs or FNs, rather it's a comparison of net benefits across different strategies. More specifically, it's the difference between the net benefit of a specific strategy and the baseline 'treat all' strategy scaled by the decision threshold and number of interventions. This gives you number of avoided unnecessary interventions by not treating all patients as the baseline strategy.
